src/main/java/axoloti/utils/Preferences.java

```diff
@@ -75,6 +78,9 @@
     @Element(required = false)
     String themePath;
 
+    @Element(required = false)
+    String Platform;
```

Comment:
Id suspect is not a difficult change... just not one I felt strongly compelled to check, but I'll take a look next time I'm looking at preferences.